### PR TITLE
In docs remove 'optional' if 'default' can be given

### PR DESCRIPTION
**Pull request by timhoffm**
## PR Summary

Following our convention to omit "optional" for parameters if a default value can be given (#15824).

### CODE REVIEW

File: lib/matplotlib/colors.py [hunk 1]
```diff
@@ -1955,7 +1955,7 @@ def blend_hsv(self, rgb, intensity, hsv_max_sat=None, hsv_max_val=None,
             An MxNx3 RGB array of floats ranging from 0 to 1 (color image).
         intensity : ndarray
             An MxNx1 array of floats ranging from 0 to 1 (grayscale image).
-        hsv_max_sat : number, optional, default: 1
+        hsv_max_sat : number, default: 1
             The maximum saturation value that the *intensity* map can shift the
             output image to. Defaults to 1.
```
Comment (QuLogic) [inline lib/matplotlib/colors.py:1960]: Can remove this:
```suggestion
            output image to.
```

File: lib/matplotlib/markers.py [hunk 2]
```diff
@@ -213,11 +213,11 @@ def __init__(self, marker=None, fillstyle=None):
...
         Parameters
         ----------
-        marker : str or array-like, optional, default: None
+        marker : str or array-like, default: None
             See the descriptions of possible markers in the module docstring.
 
-        fillstyle : str, optional, default: 'full'
-            'full', 'left", 'right', 'bottom', 'top', 'none'
+        fillstyle : str, default: 'full'
+            One of 'full', 'left", 'right', 'bottom', 'top', 'none'.
```
Comment (QuLogic) [inline lib/matplotlib/markers.py:220]: Fix quote:
```suggestion
            One of 'full', 'left', 'right', 'bottom', 'top', 'none'.
```

**Comment by timhoffm** [comment]
@story645 thanks for adding the fixes.

**Comment by story645** [comment]
@QuLogic did all the work :p

**Pull request merged**